<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundboardMenu</class>
 <widget class="QWidget" name="SoundboardMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #202020;
font-family: &quot;Cascadia Code&quot;;
border: 0px;</string>
  </property>
  <widget class="QGroupBox" name="monitor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="grooveOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>343</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/sliderGrooveOn.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="grooveOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>343</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/sliderGrooveOff.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>41</width>
      <height>403</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider { background: transparent; }

QSlider::groove:vertical {
	border: 1px solid #050505;
	border-radius: 5px;
	background-color: #000000;
	width: 11px;
	margin: 32px 32px;
}

QSlider::handle:vertical {
	image: url(:/icons/slider.png);
	height: -32px;
	margin: -32px -32px;
}
</string>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Listen to your soundboard clips&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>SOUNDBOARD MONITOR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="hotkeys">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>611</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	border: 1px solid #282828;
	border-radius: 10px;
	background: #282828;
	outline: 0;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QListWidget" name="hotkeysList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	border: 1px solid #282828;
	border-radius: 10px;
	background: #282828;
	outline: 0;
}

QListWidget::item:hover {
	background-color: #008080;
	border-radius: 10px;
}

QListWidget::item:disabled {
	background-color: transparent;
	border-radius: 10px;
}

QListWidget::item:selected {
	background-color: #006060;
	border-radius: 10px;
}
QListWidget::item:selected:!focus {
	background-color: #006060;
	border-radius: 10px;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QToolButton" name="remove">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>201</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove selected Hotkey&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
	border: 0px;
	border-right: 1px solid #000000;
	background: #404040;
	border-top-left-radius: 10px;
}

QToolButton:hover {
	background: #505050;
}

QToolButton:pressed {
	background: #484848;
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="add">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>201</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a new Hotkey&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
	border: 0px;
	border-left: 1px solid #000000;
	background: #404040;
	border-bottom-right-radius: 10px;
}

QToolButton:hover {
	background: #505050;
}

QToolButton:pressed {
	background: #484848;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="settings">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border: 1px solid #282828;
border-radius: 10px;
background: #282828;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="recordMic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>7</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggle recording of your voice&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>Record Mic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="recordSys">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>7</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggle recording of system audio, e.g. browser audio, Discord&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string>Record System Audio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadClip">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #383838;
border-radius: 10px;
}

QPushButton:hover {
background-color: #484848;
}

QPushButton:pressed {
background-color: #404040;
}</string>
    </property>
    <property name="text">
     <string>Load a Clip</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="overlap">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>7</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggle whether the sound overlaps each other, or cuts each other off&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string>Overlap</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="editor">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>611</width>
     <height>161</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #282828;
border-radius: 10px;
background: #282828;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="waveform">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>20</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #202020;
border-radius: 0px;
border: 0px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="leftCover" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>0</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 100);
border: 0px;</string>
     </property>
    </widget>
    <widget class="QWidget" name="rightCover" native="true">
     <property name="geometry">
      <rect>
       <x>461</x>
       <y>0</y>
       <width>0</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 100);
border: 0px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="startAt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>101</y>
      <width>508</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the start position of the loaded clip&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
	image: url(:/icons/marker.png);
	margin: -24px -24px;
	width: -1px;
}

QSlider::groove:horizontal {
	background: transparent;
	border: 0px;
	margin: 24px 24px;
}

QSlider {
	border: 0px;
	background: transparent;
}</string>
    </property>
    <property name="maximum">
     <number>460</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="endAt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>508</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the end position of the loaded clip&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
	image: url(:/icons/marker2.png);
	margin: -24px -24px;
	width: -1px;
}

QSlider::groove:horizontal {
	background: transparent;
	border: 0px;
	margin: 24px 24px;
}

QSlider {
	border: 0px;
	background: transparent;
}</string>
    </property>
    <property name="maximum">
     <number>460</number>
    </property>
    <property name="value">
     <number>460</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="play">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>122</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Play the selected clip&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { border-radius: 5px; }
QToolButton:hover { background-color: #404040; }
QToolButton:pressed { background-color: #303030; }</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="stop">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>122</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stop all playing clips&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { border-radius: 5px; }
QToolButton:hover { background-color: #404040; }
QToolButton:pressed { background-color: #303030; }</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>126</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clip Volume&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
image: url(:/icons/volumeHandle.png);
background: transparent;
border: 0px;
}

QSlider::groove:horizontal {
    border: 1px solid transparent;
    height: 16px; 
}

QSlider::sub-page:horizontal {
background: #00CCCC;
border-radius: 3px;
margin: 6px;
margin-left: 30px;
}

QSlider::add-page:horizontal {
background: #303030;
border-radius: 3px;
margin: 6px;
margin-right: 30px;
}</string>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="overlaySettings" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background: rgba(0,0,0,100);</string>
   </property>
  </widget>
  <widget class="QWidget" name="overlayEditor" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>611</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background: rgba(0,0,0,100);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../appicon.qrc"/>
 </resources>
 <connections/>
</ui>
